src/Layers/xrRenderGL/glSH_Texture.cpp

```diff
@@ -164,7 +164,7 @@ void CTexture::Load()
     if (nullptr == *cName)
         return;
     if (0 == xr_stricmp(*cName, "$null")) return;
-    if (nullptr != strstr(*cName, "$user$"))
+    if (0 == strncmp(*cName, "$user$", 6))
```

Comment:
Please, do the same optimization for `SH_Texture.cpp` (DX9) and `dx10SH_Texture.cpp` (DX10/11) also